tagging/routes/routes.go

```diff
@@ -56,6 +56,17 @@ func init() {
 		),
 	}.H().Register("/tagging/mostusedtags")
 
+	routes.MethodMuxer{
+		http.MethodGet: routes.H(routeGetMostUsedTagsHistory).With(
+			db.RequestTransaction{db.Db},
+			users.RequireAuthenticatedUser{users.ViewerAsParent},
+			routes.Documentation{
+				Summary:     "get most used tags",
+				Description: "Responds with most used tags for a user.",
+			},
```

Comment:
This documentation seems inaccurate, considering it's a copy-paste of the one from `/tagging/mostusedtags`